src/modules/app/constants/buildConfig.js

```diff
@@ -0,0 +1,26 @@
+// @flow
+
+import type { BuildConfigType } from '../../../../build-configs/configs/BuildConfigType'
+import integreatBuildConfig from '../../../../build-configs/configs/integreat'
+import integreatTestCmsBuildConfig from '../../../../build-configs/configs/integreat-test-cms'
+import malteBuildConfig from '../../../../build-configs/configs/malte'
+
+const buildConfigs = {
+  integreat: integreatBuildConfig,
+  'integreat-test-cms': integreatTestCmsBuildConfig,
+  malte: malteBuildConfig
+}
+
+const buildConfig = (): BuildConfigType => {
+  const buildConfigName = process.env.BUILD_CONFIG_NAME
+  if (!buildConfigName) {
+    throw Error('No BUILD_CONFIG_NAME supplied!')
+  }
+  const buildConfig: ?BuildConfigType = buildConfigs[buildConfigName]
+  if (!buildConfig) {
+    throw Error('Invalid BUILD_CONFIG_NAME supplied!')
+  }
+  return buildConfig
+}
+
+export default buildConfig
```

Comment:
Would be nice, if only the build config is included which we build for. I guess in the webapp it's solved better. Not sure, whether the technique of injecting the build-config is applicable here though.
I guess, we might just define a global __CONFIG__ object and let the bundler inject it here, but I am not sure, if that works.

Reply:
Yes, we did kind of discuss this already. I'll probably create an issue to fix this as we have a little bit time pressure :D